internal/action/command.go

```diff
@@ -539,6 +539,10 @@ func SetGlobalOptionNative(option string, nativeValue interface{}) error {
 	return config.WriteSettings(filepath.Join(config.ConfigDir, "settings.json"))
 }
 
+func ConvertRawEscFromString2Byte(rawEsc string) string {
```

Comment:
I think it should not be capitalized. Also maybe a bit shorter name?

Reply:
Shortened the name, but kept the capitalization, since it fits to the rest of the code.

Reply:
You know that in Go capitalization is not just a matter of style, right? Capitalized functions are visible to other packages. https://go.dev/ref/spec#Exported_identifiers

OTOH in micro it is in fact largely a matter of style: there are lots of exported functions that don't need to be exported. OTOH, actually for this reason I'd prefer this function to be non-capitalized: to emphasize that it is a very internal, specialized helper function that is used only by `ShowKeyCmd`, `BindCmd` and `UnbindCmd`, no one else.

BTW for the same reason, I'd put it right above `ShowKeyCmd`. Why put it between `SetGlobalOptionNative` and `SetGlobalOption`?